tests/system/test_pytorch_trainers.py

```diff
@@ -37,34 +37,13 @@ def test_simple_train(self):
 
         optimizer = nemo.backends.pytorch.actions.PtActions()
         optimizer.train(
-            tensors_to_optimize=[loss_tensor], optimizer="sgd", optimization_params={"lr": 0.0003, "num_epochs": 1},
-        )
-
-    @pytest.mark.system
-    def test_simple_train_named_output(self):
```

Comment:
Why was this test removed?

Reply:
This test was not removed. I have refactored it and changed it into unit test. Please take a look at the new file:
tests/unit/core/test_nm_tensor.py